build.gradle

```diff
@@ -50,7 +50,7 @@ ext {
     scalaVersion = scalaBinaryVersion + '.8'
     buildDir = 'build-artifacts'
     snappySparkVersion = '2.1.1.2'
-    snappyDataVersion = '1.0.1'
+    snappyDataVersion = '1.0.2-SNAPSHOT'
```

Comment:
You can change the interpreter version above, to something like '0.7.3.1' or '0.7.3.1-SNAPSHOT'

Reply:
You mean the version in the gradle file which is currently set to 0.7.3

Reply:
Yes.